samza-sql/src/main/java/org/apache/samza/sql/runner/SamzaSqlApplicationRunner.java

```diff
@@ -63,25 +69,38 @@ public SamzaSqlApplicationRunner(Boolean localRunner, Config config) {
   public static Config computeSamzaConfigs(Boolean localRunner, Config config) {
     Map<String, String> newConfig = new HashMap<>();
 
+    // TODO: Get the converter factory based on the file type. Create abstraction around this.
+    DslConverterFactory dslConverterFactory = new SamzaSqlDslConverterFactory();
```

Comment:
Looks like we are creating the DslConverter in this two places right? SamzaSQLApplication and SamzaSQLApplicationRunner. Is there a way to refactor them in a way so that we create them only once and reuse them?

Reply:
Removed creation of DSLConverter from SamzaSqlApplicationRunner. It doesn't look like it is needed in the runner. SQL EndToEnd tests passed without this.